js/util.js

```diff
@@ -93,6 +93,35 @@
         };
     }
 
+    // The find() method returns a value of the first element in the array that
+    // satisfies the provided testing function. Otherwise undefined is returned.
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find
+    if (!Array.prototype.find) {
+      Object.defineProperty(Array.prototype, 'find', {
+        value: function(predicate) {
+         'use strict';
```

Comment:
In theory this file should be jshintable, although I think our current grunt file doesn't include silk. Please drop the `'use strict'` b/c it's already at the top of the file, and add `/* jshint bitwise: false */` to the top of the function.